<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1515</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1496</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Выберите тип диаграммы</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>BarChart</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PieChart</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>чёрно-белый график</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="palette">
          <palette>
           <active/>
           <inactive/>
           <disabled/>
          </palette>
         </property>
         <property name="text">
          <string>Печать графика</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="horizontalFrame_3">
         <property name="minimumSize">
          <size>
           <width>1000</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1515</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPrint">
    <property name="title">
     <string>PrintChart</string>
    </property>
   </widget>
   <addaction name="menuPrint"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
